### PR TITLE
Bf/zenko 436 pick read location fix

### PR DESCRIPTION
**Pull request by alexanderchan-scality**
(cherry picked from commit b7fb379)

### CODE REVIEW

File: lib/api/apiUtils/object/checkReadLocation.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/**
+ * checkReadLocation - verify that a bucket's default read location exists
+ * a specified read data locator
```
Comment (jonathan-gramain) [inline lib/api/apiUtils/object/checkReadLocation.js]: typo `exists [for] a specified ...`

File: lib/api/apiUtils/object/checkReadLocation.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * @param {Config} config - Config object
+ * @param {string} locationName - location constraint
+ * @param {string} objectKey - object key
+ * @param {string} bucketName - bucket name
+ * @return {Object | null} return object if location exists; otherwise, null
```
Comment (jonathan-gramain) [inline lib/api/apiUtils/object/checkReadLocation.js]: minor: I suggest a more descriptive comment about the returned value, e.g. returns the data storage location?

**Review by jonathan-gramain — DISMISSED** [review]

**Review by philipyoo — APPROVED** [review]

File: lib/api/apiUtils/object/checkReadLocation.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+/**
+ * checkReadLocation - verify that a bucket's default read location exists
+ * a for specified read data locator
```
Comment (philipyoo) [inline lib/api/apiUtils/object/checkReadLocation.js:3]: minor: for a

**Review by jonathan-gramain — APPROVED** [review]

**Comment by alexanderchan-scality** [comment]
closed in favor of #1272

**Pull request closed without merging**